packages/app/src/components/commons/PermissionItem.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import { Grid } from "@mui/material"
+import { styled } from "@mui/styles"
+import { palette } from "../../theme"
+import { Permission } from "../../models/publication"
+import EditIcon from "@mui/icons-material/Edit"
+import { WalletBadge } from "./WalletBadge"
+
+const PermissionItemContainer = styled(Grid)({
+  minHeight: "48px",
+  background: palette.grays[100],
+  borderRadius: 4,
+  padding: "10px 20px",
+  cursor: "pointer",
+  "&:hover": {
+    background: palette.grays[200],
+  },
+})
+const PermissionItemIconGrid = styled(Grid)({
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  color: "#91908e",
+})
+
+const PermissionItemEditContainer = styled(Grid)({
+  border: `2px solid ${palette.grays[600]}`,
+  background: palette.whites[600],
+  borderRadius: 4,
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  width: 32,
+  height: 32,
+  cursor: "pointer",
+})
+
+type PermissionItemProps = {
+  permission: Permission
+  onClick: (id: string) => void
+}
+const PermissionItem: React.FC<PermissionItemProps> = ({ permission, onClick }) => {
+  const { address, id } = permission
+
+  return (
+    <PermissionItemContainer container alignItems={"center"} onClick={() => onClick(id ? id : "")}>
+      <Grid item xs={10}>
+        <WalletBadge address={address} />
+      </Grid>
+      <PermissionItemIconGrid item xs={2}>
+        <PermissionItemEditContainer>
+          <EditIcon style={{ color: palette.grays[1000] }} />
```

Comment:
We should probably not show the edit button here to users that do not have the "Can edit the publication permissions" / `publication/permissions`

Reply:
Done 🚀

<img width="1920" alt="image" src="https://user-images.githubusercontent.com/19823989/163181850-74a756fc-d7b0-4fde-8603-b341fd74f97e.png">